pkg/kubelet/cm/container_manager.go

```diff
@@ -25,7 +25,7 @@ type ContainerManager interface {
 	// Runs the container manager's housekeeping.
 	// - Ensures that the Docker daemon is in a container.
 	// - Creates the system container where all non-containerized processes run.
-	Start() error
+	Start(*api.Node) error
```

Comment:
Yes! I think that would be more appropriate here. I would have a no-op interface for now.